src/Message/RefundResponse.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+final class RefundResponse extends AbstractResponse implements ResponseInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function isSuccessful()
+    {
+        return true;
```

Comment:
Shouldn't there be a check on some kind of error response code here to verify if the API call was successful?